whipper/command/cd.py

```diff
@@ -253,6 +253,12 @@ def add_arguments(self):
                                  "if the patched cdparanoia package is "
                                  "installed and the drive "
                                  "supports this feature. ")
+        self.parser.add_argument('--exclude-tracks',
+                                 action="store", dest="excluded_tracks",
+                                 default="",
+                                 help="Comma separated list of tracks to "
+                                 "exclude. This will propably mess up "
```

Comment:
I think it'd be best to handle the logging situation as part of implementing this feature. We'd like to avoid having incompatible features butting heads.